site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -62,12 +62,27 @@ The type of security protocol FusionAuth will use when connecting to the SMTP se
 [field]#{base_field_name}.emailConfiguration.setPasswordEmailTemplateId# [type]#[UUID]#::
 The Id of the Email Template that is used when a user had their account created for them and they must set their password manually and they are sent an email to set their password.
 
+[field]#tenant.emailConfiguration.unverified.allowEmailChangeWhenGated# [type]#[Boolean]# [since]#Available since 1.27.0#::
```

Comment:
```suggestion
[field]#{base_field_name}.emailConfiguration.unverified.allowEmailChangeWhenGated# [type]#[Boolean]# [since]#Available since 1.27.0#::
```